vendor/bat-native-ledger/src/bat/ledger/internal/media/vimeo.cc

```diff
@@ -649,12 +672,16 @@ void Vimeo::OnPublisherVideoPage(
   const uint64_t duration = GetDuration(old_event, event_info);
   events[media_key] = event_info;
 
+  const std::string publisher_favicon = GenerateFaviconUrl(response);
+
   SavePublisherInfo(media_key,
-                    duration,
-                    user_id,
-                    GetNameFromVideoPage(response),
-                    GetUrlFromVideoPage(response),
-                    0);
+                  duration,
```

Comment:
alignment is off. It should be like this:

```
SavePublisherInfo(
    media_key,
    duration,
    .....);
```

Reply:
Fixed